### PR TITLE
Add lock support on maintenances commands

### PR DESCRIPTION
**Pull request by kuzmany**
| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 

#### Description:
This PR added lock for 
- `mautic:maintenance:cleanup`
- `mautic:unusedip:delete`

It prevent run same command twice or more times.

#### Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2.  Run  command and try it run twice in same time

- `mautic:maintenance:cleanup`
- `mautic:unusedip:delete`

3. You should see `Script in progress. Can force execution by using --bypass-locking` on second call.

### CODE REVIEW

**Review by escopecz — DISMISSED** [review]
Code changes look great 👍  Thanks!

**Comment by escopecz** [comment]
@kuzmany can you please add tests so the coverage is not decreased?

**Review by jos0405 — DISMISSED** [review]
Didn't work for me in Gitpod, but I think it's a different reason.
On my local and on another instance it works as described.

**Comment by RCheesley** [comment]
@kuzmany some tests to fix here as well as needing to add coverage (also some conflicts to resolve) :)

**Comment by mabumusa1** [comment]
@kuzmany can you resolve the conflict so we can merge it today?

**Comment by kuzmany** [comment]
@mabumusa1 conflicts resolved, all tests passed

File: app/bundles/CoreBundle/Tests/Command/CleanupMaintenanceCommandTest.php [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    public function testCleanupMaintenanceCommand(): void
+    {
+        $lead = new Lead();
+        $lead->setLastActive(new \DateTime('-1 year'));
+        $this->em->persist($lead);
+        $this->em->flush();
+
+        $contactId = $lead->getId();
+
+        // Delete unused IP address.
```
Comment (escopecz) [inline app/bundles/CoreBundle/Tests/Command/CleanupMaintenanceCommandTest.php]: It's deleting contacts older than 180 days, right? No IP addresses.

**Review by escopecz — APPROVED** [review]
Thanks!

**Pull request merged**